docker/postgres/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN apt-get update && apt-get install -y \
     git
 
 RUN cd /extensions/postgres-json-schema && make install && make installcheck
```

Comment:
@mattkrick could we remove this, and all the files related to it, as we are not really using it anywhere?

Reply:
Just remove it and see what happens?

Reply:
It should work no trouble because I'm not adding that in any other environment, even when I'm building a Docker image for us. I'm just asking in case he wants to keep it for any reason that I cannot see. IMHO we should just remove it, and all files with it.

Reply:
Removed!